### PR TITLE
Update zh-CN.yml

### PR DESCRIPTION
**Pull request by 2016WUJI01**
Update zh-CN.yml.
Maybe it need @notrichardpeng (Richard Jiankun Peng) to review those translations.

### CODE REVIEW

**Comment by gregorbg** [comment]
> Maybe it need @notrichardpeng (Richard Jiankun Peng) to review those translations.

Thanks for your submission! If you can get any Chinese-speaking Delegate to vouch for you, we can add you to the list of verified translators and automatically merge any submissions you make

File: WcaOnRails/config/locales/zh-CN.yml [hunk 1]
```diff
@@ -2507,15 +2535,87 @@ zh-CN:
...
+    #original_hash: cdab446
+    who_we_are_title: 我们是谁
+    #original_hash: c0d9a3e
+    who_we_are_content_html: >-
+      <p>世界魔方协会负责管理魔方的比赛，这些魔方是通过转动一部分块来操作，通常称为“魔方”（twisty puzzles)。这些魔方中最著名的是三阶魔方，由匈牙利的鲁比克教授发明。这些魔方中的一部分被选为WCA的官方赛事项目。</p>
+      <p>随着WCA在过去十年的发展，已有超过200,000人参加了我们的比赛。尽管有这样的增长，我们仍然是一个几乎完全由志愿者运营的组织，从当地组织者到WCA代表，甚至WCA董事会。我们非常感谢所有抽出时间为WCA做出贡献的人。有了更多的财政支持后，我们希望在认可全球各地比赛的同时，能够为我们的比赛提升到更高水平的层次。</p>
+    #original_hash: bccd8c5
+    spirit: 我们的精神是：来自世界各地的人们在友好的气氛中一起玩的开心，互相帮助，并表现出运动员的精神。
+    #original_hash: 69131cc
+    spirit_footer: WCA精神
```
Comment (notrichardpeng) [inline WcaOnRails/config/locales/zh-CN.yml:2551]: “WCA精神” 或 “WCA的精神” 都可以，但是感觉最好和上面的 "WCA的使命" 保持一样（用不用 “的”）

File: WcaOnRails/config/locales/zh-CN.yml [hunk 2]
```diff
@@ -2590,6 +2699,16 @@ zh-CN:
     title: 文档
     #original_hash: 1b537dc
     determination_letter: 国税局的裁定书
+    #original_hash: 9855f80
+    bylaws: 内部章程
+    #original_hash: fdceb44
+    motions: 提议
```
Comment (notrichardpeng) [inline WcaOnRails/config/locales/zh-CN.yml:2705]: “议案” 可能好一点

**Review by notrichardpeng — CHANGES_REQUESTED** [review]

File: WcaOnRails/config/locales/zh-CN.yml (hunk 2 above)
Comment (2016WUJI01) [reply WcaOnRails/config/locales/zh-CN.yml]: Changed

Comment (2016WUJI01) [reply WcaOnRails/config/locales/zh-CN.yml]: And I only changed two chinese words, But Ruby tests shows "All checks have failed".

File: WcaOnRails/config/locales/zh-CN.yml [hunk 3]
```diff
@@ -1170,6 +1196,8 @@ zh-CN:
     about_us:
       #original_hash: 3ffb811
       title: 关于我们
+      #original_hash: ed77e4c
+      text: 世界魔方协会负责管理魔方的比赛，这些魔方是通过转动一部分块来操作，通常称为“魔方”（twisty puzzles)。
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:1200]: Using `一部分块` feels a little bit unnatural to me.
Is there a better expression for this?

Comment (2016WUJI01) [reply WcaOnRails/config/locales/zh-CN.yml:1200]: Maybe 转动魔方的一部分来操作 ?

File: WcaOnRails/config/locales/zh-CN.yml [hunk 4]
```diff
@@ -2507,15 +2535,87 @@ zh-CN:
   about:
     #original_hash: 9ac918a
     title: 关于WCA
+    #original_hash: af00571
+    mission_statement: 我们的使命是：在公平、平等的条件下，在更多国家举办更多、更多人、更有趣的比赛。
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:2539]: `在更多国家举办更多、更多人、更有趣的比赛。`
有点不通顺，不顺口

I think `more competitions in more countries with more people and more fun` can be expressed better with `在更多国家举办更多更有趣的比赛，并让更多人参与。`

File: WcaOnRails/config/locales/zh-CN.yml [hunk 5]
```diff
@@ -2507,15 +2535,87 @@ zh-CN:
...
+    #original_hash: e627278
+    mission_footer: WCA的使命
+    #original_hash: cdab446
+    who_we_are_title: 我们是谁
+    #original_hash: c0d9a3e
+    who_we_are_content_html: >-
+      <p>世界魔方协会负责管理魔方的比赛，这些魔方是通过转动一部分块来操作，通常称为“魔方”（twisty puzzles)。这些魔方中最著名的是三阶魔方，由匈牙利的鲁比克教授发明。这些魔方中的一部分被选为WCA的官方赛事项目。</p>
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:2546]: same as hash: ed77e4c `一部分块`

File: WcaOnRails/config/locales/zh-CN.yml [hunk 6]
```diff
@@ -2507,15 +2535,87 @@ zh-CN:
...
+    spirit_footer: WCA的精神
+    #original_hash: c1781ee
+    our_goals_title: 我们的目标
+    #original_hash: 743ca74
+    our_goals_content_html: >-
+      <p>世界魔方协会希望通过增加魔方比赛，为世界各地的年轻人带来参与社群和领导社群的机会。我们相信，比赛和社群内的经历为参与其中的年轻人创造了成长机会，让更多人群接触到魔方，让魔友群体茁壮成长。关于董事会对于WCA愿景与策略的文件可以在<a href="https://www.worldcubeassociation.org/documents/WCA_Vision_and_Strategy.pdf">这里</a>查阅。</p>
+    #original_hash: dc71704
+    world_championship_title: 世界锦标赛
+    #original_hash: 57cf7f7
+    world_championship_content_html: '<p>每两年一次，我们将在比赛中决定谁是世界冠军。这些活动需要大量志愿者进行极其仔细的规划，并需要投入大量资金来预订场地并进行必要的准备。</p>'
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:2560]: `sanction` 没有翻译到

File: WcaOnRails/config/locales/zh-CN.yml [hunk 7]
```diff
@@ -2507,15 +2535,87 @@ zh-CN:
...
+    #original_hash: 1aa5344
+    contributing_title: 对我们进行赞助
+    #original_hash: 538a82f
+    contributing_content_html: '您可以使用信用卡在Guidestar %{here_credit}或是Paypal %{here_paypal}平台捐款。如果您希望使用支票，请寄到：'
+    #original_hash: 2af9b71
+    tax_deductible_note: 在美国，捐款可以免税。
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:2566]: tax-deductible 与 tax-free 不同
应该是`抵税`

File: WcaOnRails/config/locales/zh-CN.yml [hunk 8]
```diff
@@ -2507,15 +2535,87 @@ zh-CN:
...
+    donation_banner:
+      #original_hash: 1aaf818
+      content: >-
+        世界魔方协会是一个 501(c)(3) 免税组织。 作为一个 100% 由志愿者主导的非营利组织，我们欢迎您的捐赠，以帮助我们进一步实现在世界各地举办更有趣的比赛的愿景。
+      #original_hash: 24c87db
+      donate_credit: 用信用卡捐款
+      #original_hash: 0e5a9c3
+      donate_paypal: 用Paypal捐款
+      #original_hash: 96d77f3
+      donate_other: 其他方式捐款
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:2581]: Other ways to help is non-financial aid.

It should be 
`其他方式的协助` or `其他方式的帮助`

File: WcaOnRails/config/locales/zh-CN.yml [hunk 9]
```diff
@@ -2562,6 +2662,11 @@ zh-CN:
         name: WCA比赛公示组
         #original_hash: f511502
         description: 该组负责审核和公示WCA比赛，并确保此类比赛符合WCA标准。
+      wmt:
+        #original_hash: 2035811
+        name: WCA市场组
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:2667]: Personally, I prefer `市场营销组` or `营销组` 
Just `市场组` 怪怪的

File: WcaOnRails/config/locales/zh-CN.yml [hunk 10]
```diff
@@ -2573,11 +2678,15 @@ zh-CN:
         #original_hash: 8f888d6
         description: 该团队负责监督和协助WCA成为一个国际认可的竞技组织。
       wat:
+        #original_hash: 90a1280
+        name: WCA资料整理组
         #original_hash: fae6850
-        info_html: WCA资料整理组是一个WCA的咨询委员会。这个团队的任务是协助并负责将WCA过去的资料作整理保存。
+        description: WCA资料整理组是一个WCA的咨询委员会。这个团队的任务是协助并负责将WCA过去的资料作整理保存。
```
Comment (hanwu85) [inline WcaOnRails/config/locales/zh-CN.yml:2684]:
```suggestion
WCA资料整理组是一个WCA的咨询委员会。这个团队的任务是协助并负责将WCA过去的资料作整理保存。
```

File: WcaOnRails/config/locales/zh-CN.yml (hunk 3 above)
Comment (hanwu85) [reply WcaOnRails/config/locales/zh-CN.yml:1200]: 感觉上比现在的好

**Comment by hanwu85** [comment]
You can refer to this guideline for easier and faster translation for WCA website. 
https://docs.worldcubeassociation.org/guides/translating_the_website

**Review by hanwu85 — APPROVED** [review]
I think this is not a bad translation.
No translation is perfect, but I think it is good enough.
@notrichardpeng would be appreciated if you could give your input as well.

**Comment by notrichardpeng** [comment]
LGTM, sorry for the delay, and thanks for your additional input @hanwu85

**Review by notrichardpeng — APPROVED** [review]

**Pull request merged**